src/styles/index.scss

```diff
@@ -1,3 +1,204 @@
+$title-color: #060b35;
+$txt-color: #616070;
+
+html {
+  font-family: Roboto, serif;
+  font-weight: 400, 500, 700;
```

Comment:
The `font-weight` property should not have multiple values like `400, 500, 700`. Instead, specify a single value or use separate rules for different elements if needed.